src/templates/aws/advanced.ts

```diff
@@ -112,51 +111,190 @@ export default class Advanced {
 
     const ssmModuleContent = dedent`
     module "ssm" {
-      source = "../modules/ssm"
-    
+      source = "./modules/ssm"
+
       namespace = var.namespace
       secret_key_base       = var.secret_key_base
-    
+
       aws_access_key_id     = var.aws_access_key_id
       aws_secret_access_key = var.aws_secret_access_key
-    
+
       rds_username      = var.rds_username
       rds_password      = var.rds_password
       rds_database_name = var.rds_database_name
       rds_endpoint      = module.db.db_endpoint
     }`
 
-    injectToFile('main.tf', ssmModuleContent, this.options, {insertAfter: '# SSM'})
+    injectToFile('main.tf', ssmModuleContent, this.options, {
+      insertAfter: '# SSM',
+    })
   }
 
   private applySecurityGroup(): void {
-    copyDir('aws/modules/security_group', 'modules/security_group', this.options)
+    copyDir(
+      'aws/modules/security_group',
+      'modules/security_group',
+      this.options,
+    )
 
     const ssmVariablesContent = dedent`
```

Comment:
due to copy-paste from the previous step, I used the wrong variable name here, could you please help me to fix it 🙏
```suggestion
    const securityGroupVariablesContent = dedent`
```